editor/scene/3d/node_3d_editor_plugin.cpp

```diff
@@ -5674,7 +5710,7 @@ void Node3DEditorViewport::update_transform(bool p_shift) {
 			// TRANSLATORS: Refers to changing the scale of a node in the 3D editor.
 			set_message(TTR("Scaling:") + " (" + String::num(motion_snapped.x, snap_step_decimals) + ", " +
 					String::num(motion_snapped.y, snap_step_decimals) + ", " + String::num(motion_snapped.z, snap_step_decimals) + ")");
-			if (local_coords) {
+			if (spatial_editor->are_local_coords_enabled() && _edit.plane != TRANSFORM_VIEW) {
```

Comment:
Why did you change the original code? The original code defined local_coords in the line above for reuse and I think it's equivalent.